### ISSUE TITLE
maxuploadtarget=1000 to be optional, this is important for bitcoin health

### ISSUE THREAD

**Issue by tukac** [mynodebtc/mynode#715]
Default line in myNode bitcoin config is maxuploadtarget=1000. This line will force bitcoind to target daily upload to 1000 Mb. This is about 11.5 kb/s. Not much at all! Many people have fast upload speed, so allowing this to be altered and/or disabled seems like more than good idea.

This is important for bitcoin network health and censorship resistance. Jameson Loop had made a paper half a year ago documenting network bottlenecks in syncing download speed (without fast sync). He is periodically benchmarking various bitcoin clients and their initial sync times. Having previous syncing times he concluded that initial download time is not getting faster but probably opposite. Now with I2P everybody with fast upload speed can be seeding node.

(https://blog.lopp.net/is-bitcoin-network-slowing-down/)

**Comment by tehelsper** [comment]
Yeah, I actually messaged lopp after seeing his article and was worried about he same thing. It could possible cause peer churn during IBD if everyone's 1000MB limit gets used up. I plan on making this configurable in a future update.
